library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -1121,6 +1123,7 @@ private void setRangeDates(CalendarDay min, CalendarDay max) {
         boolean dynamicHeightEnabled = false;
         CalendarMode calendarMode = CalendarMode.MONTHS;
         CalendarDay currentMonth = null;
+        boolean saveCurrentPosition;
```

Comment:
This is an incredibly small change that would be very useful for any time the view is reloaded while in use.